time: 2020-05-05 05:06:26.973387Z
tags: worker-0
test: tempest.api.workloadmgr.test_cleaner.WorkloadTest.test_cleaner[id-9fe07175-912e-49a5-a629-5f52eeada4c9]
time: 2020-05-05 05:06:33.849968Z
successful: tempest.api.workloadmgr.test_cleaner.WorkloadTest.test_cleaner[id-9fe07175-912e-49a5-a629-5f52eeada4c9] [ multipart
Content-Type: text/plain;charset=utf8
pythonlogging:''
F503
2020-05-05 10:36:28,598 22148 DEBUG    [tempest.api.workloadmgr.test_cleaner] 
restores deleted

2020-05-05 10:36:28,741 22148 DEBUG    [tempest.api.workloadmgr.test_cleaner] 
snapshots deleted

2020-05-05 10:36:28,918 22148 DEBUG    [tempest.api.workloadmgr.test_cleaner] 
workloads deleted

2020-05-05 10:36:29,322 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 201 POST https://traincentos.triliodata.demo:5000/v3/auth/tokens
2020-05-05 10:36:29,323 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json'}
        Body: <omitted>
    Response - Headers: {'status': '201', 'content-length': '6272', 'x-xss-protection': '1; mode=block', 'content-security-policy': "default-src 'self' https: wss:;", 'x-content-type-options': 'nosniff', 'x-subject-token': 'gAAAAABesPRVjSVJT2ha0J1XjDf61mZuj0jZaLUa3m-0XWsePSQrmVzQrwFcAZpI6fjS21NhtcuOMBAl_uO65LyHsV9r3kCLDSquYMluI7_1xuLg_NJe9LPBT03zQ63VkPms_XiUcTD-iVdZfKSotcgA56MRDssVu0qDr3APDmFY_dYRhm7HefQ', 'vary': 'X-Auth-Token', 'server': 'nginx/1.16.1', 'connection': 'close', 'date': 'Tue, 05 May 2020 05:06:29 GMT', 'x-frame-options': 'DENY', 'content-type': 'application/json', 'x-openstack-request-id': 'req-b48ff273-1e52-4805-aae3-8f32be0381d7'}
        Body: {"token": {"is_domain": false, "methods": ["password"], "roles": [{"id": "0a4011fc83a54d24871e469d39753411", "name": "_member_"}, {"id": "6fbc42878c844c42b85ddee79ae4766e", "name": "admin"}, {"id": "cda93355d59e4860a201582c6cdd5cda", "name": "reader"}, {"id": "802f472b8aff4aefa2e6c46e7eec6a04", "name": "member"}], "expires_at": "2020-05-05T17:06:29.000000Z", "project": {"domain": {"id": "88a4764ba8e7477ea28cf345bc36ffa2", "name": "clouddomain"}, "id": "487d69e3dac6411ba4ffca3a5ac61738", "name": "tproject"}, "catalog": [{"endpoints": [{"url": "http://172.26.0.32:8776/v2/487d69e3dac6411ba4ffca3a5ac61738", "interface": "internal", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "7118453d6bfe4dfcb91cde15feaca714"}, {"url": "https://traincentos.triliodata.demo:8776/v2/487d69e3dac6411ba4ffca3a5ac61738", "interface": "public", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "71bc950432c5416ab4eb197f24acef3f"}, {"url": "http://172.26.0.32:8776/v2/487d69e3dac6411ba4ffca3a5ac61738", "interface": "admin", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "a6b8a613dbd84af7acfddb90bd213604"}], "type": "volumev2", "id": "298d35249d3949259e51ec5fdecb78ed", "name": "cinderv2"}, {"endpoints": [{"url": "http://172.26.0.32:8774/v2.1", "interface": "admin", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "4a3ab2583e804bf3853201e790d2636a"}, {"url": "http://172.26.0.32:8774/v2.1", "interface": "internal", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "a3d0a91a938d4edb8c1be7cb3ce82922"}, {"url": "https://traincentos.triliodata.demo:8774/v2.1", "interface": "public", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "c530b661e10e4310b6b502e383b197a8"}], "type": "compute", "id": "55817385a6d547e4b5d95b5f66038cea", "name": "nova"}, {"endpoints": [{"url": "https://traincentos.triliodata.demo:9696", "interface": "public", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "10189c5f84a4403ca1f270fe98e21699"}, {"url": "http://172.26.0.32:9696", "interface": "admin", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "98e351fa96d44d6ba151513d211ab213"}, {"url": "http://172.26.0.32:9696", "interface": "internal", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "da4c50885aff497b93e74e5f1ac6adf9"}], "type": "network", "id": "6b907ed507b142aab8aad9e56fcaf18d", "name": "neutron"}, {"endpoints": [{"url": "http://172.26.0.32:8780", "interface": "admin", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "20c3abff38574a989564819b192d6f9b"}, {"url": "https://traincentos.triliodata.demo:8780", "interface": "public", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "3ae6894c74334f17b15781de8f8fa35e"}, {"url": "http://172.26.0.32:8780", "interface": "internal", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "ac9b089c58cb4de98b083bdcd3a0b8c9"}], "type": "placement", "id": "73fddeba3eb74c4994bf0222e1829d15", "name": "placement"}, {"endpoints": [{"url": "https://osavip-nc-168.triliodata.demo:8780/v1/487d69e3dac6411ba4ffca3a5ac61738", "interface": "public", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "2ad429cef430444e803e09a8195da54f"}, {"url": "https://osavip-nc-168.triliodata.demo:8780/v1/487d69e3dac6411ba4ffca3a5ac61738", "interface": "internal", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "549452e72bbd4024baab9975a32ad39f"}, {"url": "https://osavip-nc-168.triliodata.demo:8780/v1/487d69e3dac6411ba4ffca3a5ac61738", "interface": "admin", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "bfbff9d21814407ca2cfa978ab95ed33"}], "type": "workloads", "id": "9e996ed9bd574f94adef3373b1053fe0", "name": "TrilioVaultWLM"}, {"endpoints": [{"url": "http://172.26.0.32:8784/v2", "interface": "internal", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "25516677760d489c9d6a44c6979b7d4f"}, {"url": "https://traincentos.triliodata.demo:8784/v2", "interface": "public", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "70759a7e2e3344099735c6789914bfe4"}, {"url": "http://172.26.0.32:8784/v2", "interface": "admin", "region": "US-WEST-2", "region_id": "US-WEST-2", "id": "a2d998b
2020-05-05 10:36:29,893 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/servers 0.569s
2020-05-05 10:36:29,894 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '1963', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers', 'x-compute-request-id': 'req-210b596d-c4d7-4f70-9c92-1b728dcc30b3', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-210b596d-c4d7-4f70-9c92-1b728dcc30b3'}
        Body: {"servers": [{"id": "82b3b4c9-7621-442e-b720-18bbda35f27d", "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/82b3b4c9-7621-442e-b720-18bbda35f27d", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/82b3b4c9-7621-442e-b720-18bbda35f27d", "rel": "bookmark"}], "name": "tempest_test_vm_1"}, {"id": "8276bbc6-621a-44c8-9a44-6a14e726e1d1", "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1", "rel": "bookmark"}], "name": "tempest_test_vm_1"}, {"id": "1bda2d0b-c2e2-4933-8772-5f2b0b68ec45", "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45", "rel": "bookmark"}], "name": "tempest_test_vm_1"}, {"id": "8ffb8122-3d97-42ff-886f-7b437954d3d6", "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6", "rel": "bookmark"}], "name": "tempest_test_vm_1"}, {"id": "7d4e4e4b-4729-4603-88ac-8ccf0794e492", "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492", "rel": "bookmark"}], "name": "tempest_test_vm_1"}, {"id": "297fffd4-c2a9-4e24-8a93-97efbb3f544c", "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c", "rel": "bookmark"}], "name": "tempest_test_vm_1"}]}
2020-05-05 10:36:30,436 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/servers/82b3b4c9-7621-442e-b720-18bbda35f27d 0.534s
2020-05-05 10:36:30,437 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/82b3b4c9-7621-442e-b720-18bbda35f27d', 'x-compute-request-id': 'req-429b5b51-1035-4e41-9264-0b11d419fba5', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-429b5b51-1035-4e41-9264-0b11d419fba5'}
        Body: {"server": {"OS-EXT-STS:task_state": null, "addresses": {}, "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/82b3b4c9-7621-442e-b720-18bbda35f27d", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/82b3b4c9-7621-442e-b720-18bbda35f27d", "rel": "bookmark"}], "image": {"id": "bc2fd9b7-b114-4c75-8350-69aa90223793", "links": [{"href": "https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793", "rel": "bookmark"}]}, "OS-EXT-STS:vm_state": "error", "OS-EXT-SRV-ATTR:instance_name": "instance-00000012", "OS-SRV-USG:launched_at": null, "flavor": {"id": "dd8c9755-a198-4ef8-bb97-e9b8f15f765d", "links": [{"href": "https://traincentos.triliodata.demo:8774/flavors/dd8c9755-a198-4ef8-bb97-e9b8f15f765d", "rel": "bookmark"}]}, "id": "82b3b4c9-7621-442e-b720-18bbda35f27d", "user_id": "1aa40866a8ab4c8fba95febee63f4b3f", "OS-DCF:diskConfig": "MANUAL", "accessIPv4": "", "accessIPv6": "", "OS-EXT-STS:power_state": 0, "OS-EXT-AZ:availability_zone": "nova", "config_drive": "", "status": "ERROR", "updated": "2020-05-04T13:40:11Z", "hostId": "", "OS-EXT-SRV-ATTR:host": null, "OS-SRV-USG:terminated_at": null, "key_name": "tempest_test_key_pair", "OS-EXT-SRV-ATTR:hypervisor_hostname": null, "name": "tempest_test_vm_1", "created": "2020-05-04T13:40:10Z", "tenant_id": "487d69e3dac6411ba4ffca3a5ac61738", "os-extended-volumes:volumes_attached": [], "fault": {"message": "No valid host was found. ", "code": 500, "details": "Traceback (most recent call last):\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py\", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py\", line 160, in select_destinations\n    return cctxt.call(ctxt, 'select_destinations', **msg_args)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py\", line 181, in call\n    transport_options=self.transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py\", line 129, in _send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 646, in send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py\", line 235, in inner\n    return func(*args, **kwargs)\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py\", line 199, in select_destinations\n    raise exception.NoValidHost(reason=\"\")\n\nNoValidHost: No valid host was found. \n\n", "created": "2020-05-04T13:40:11Z"}, "metadata": {}}}
2020-05-05 10:36:30,471 22148 DEBUG    [tempest.api.workloadmgr.base] Vm details :response: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/82b3b4c9-7621-442e-b720-18bbda35f27d', 'x-compute-request-id': 'req-429b5b51-1035-4e41-9264-0b11d419fba5', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-429b5b51-1035-4e41-9264-0b11d419fba5'}
Body: {u'server': {u'OS-EXT-STS:task_state': None, u'addresses': {}, u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/v2.1/servers/82b3b4c9-7621-442e-b720-18bbda35f27d', u'rel': u'self'}, {u'href': u'https://traincentos.triliodata.demo:8774/servers/82b3b4c9-7621-442e-b720-18bbda35f27d', u'rel': u'bookmark'}], u'image': {u'id': u'bc2fd9b7-b114-4c75-8350-69aa90223793', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793', u'rel': u'bookmark'}]}, u'OS-EXT-STS:vm_state': u'error', u'OS-EXT-SRV-ATTR:instance_name': u'instance-00000012', u'OS-SRV-USG:launched_at': None, u'flavor': {u'id': u'dd8c9755-a198-4ef8-bb97-e9b8f15f765d', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/flavors/dd8c9755-a198-4ef8-bb97-e9b8f15f765d', u'rel': u'bookmark'}]}, u'id': u'82b3b4c9-7621-442e-b720-18bbda35f27d', u'user_id': u'1aa40866a8ab4c8fba95febee63f4b3f', u'OS-DCF:diskConfig': u'MANUAL', u'accessIPv4': u'', u'accessIPv6': u'', u'OS-EXT-STS:power_state': 0, u'OS-EXT-AZ:availability_zone': u'nova', u'config_drive': u'', u'status': u'ERROR', u'updated': u'2020-05-04T13:40:11Z', u'hostId': u'', u'OS-EXT-SRV-ATTR:host': None, u'OS-SRV-USG:terminated_at': None, u'key_name': u'tempest_test_key_pair', u'OS-EXT-SRV-ATTR:hypervisor_hostname': None, u'name': u'tempest_test_vm_1', u'created': u'2020-05-04T13:40:10Z', u'tenant_id': u'487d69e3dac6411ba4ffca3a5ac61738', u'os-extended-volumes:volumes_attached': [], u'fault': {u'message': u'No valid host was found. ', u'code': 500, u'details': u'Traceback (most recent call last):\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py", line 160, in select_destinations\n    return cctxt.call(ctxt, \'select_destinations\', **msg_args)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py", line 181, in call\n    transport_options=self.transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py", line 129, in _send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 646, in send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py", line 235, in inner\n    return func(*args, **kwargs)\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py", line 199, in select_destinations\n    raise exception.NoValidHost(reason="")\n\nNoValidHost: No valid host was found. \n\n', u'created': u'2020-05-04T13:40:11Z'}, u'metadata': {}}}
2020-05-05 10:36:30,924 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1 0.442s
2020-05-05 10:36:30,925 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1', 'x-compute-request-id': 'req-74e5f154-5c6a-401a-8e96-3f60739407fd', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-74e5f154-5c6a-401a-8e96-3f60739407fd'}
        Body: {"server": {"OS-EXT-STS:task_state": null, "addresses": {}, "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1", "rel": "bookmark"}], "image": {"id": "bc2fd9b7-b114-4c75-8350-69aa90223793", "links": [{"href": "https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793", "rel": "bookmark"}]}, "OS-EXT-STS:vm_state": "error", "OS-EXT-SRV-ATTR:instance_name": "instance-00000011", "OS-SRV-USG:launched_at": null, "flavor": {"id": "9265bb6c-4fe7-4dda-a974-1886be221574", "links": [{"href": "https://traincentos.triliodata.demo:8774/flavors/9265bb6c-4fe7-4dda-a974-1886be221574", "rel": "bookmark"}]}, "id": "8276bbc6-621a-44c8-9a44-6a14e726e1d1", "user_id": "1aa40866a8ab4c8fba95febee63f4b3f", "OS-DCF:diskConfig": "MANUAL", "accessIPv4": "", "accessIPv6": "", "OS-EXT-STS:power_state": 0, "OS-EXT-AZ:availability_zone": "nova", "config_drive": "", "status": "ERROR", "updated": "2020-05-04T13:39:29Z", "hostId": "", "OS-EXT-SRV-ATTR:host": null, "OS-SRV-USG:terminated_at": null, "key_name": "tempest_test_key_pair", "OS-EXT-SRV-ATTR:hypervisor_hostname": null, "name": "tempest_test_vm_1", "created": "2020-05-04T13:39:28Z", "tenant_id": "487d69e3dac6411ba4ffca3a5ac61738", "os-extended-volumes:volumes_attached": [], "fault": {"message": "No valid host was found. ", "code": 500, "details": "Traceback (most recent call last):\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py\", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py\", line 160, in select_destinations\n    return cctxt.call(ctxt, 'select_destinations', **msg_args)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py\", line 181, in call\n    transport_options=self.transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py\", line 129, in _send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 646, in send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py\", line 235, in inner\n    return func(*args, **kwargs)\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py\", line 199, in select_destinations\n    raise exception.NoValidHost(reason=\"\")\n\nNoValidHost: No valid host was found. \n\n", "created": "2020-05-04T13:39:29Z"}, "metadata": {}}}
2020-05-05 10:36:30,933 22148 DEBUG    [tempest.api.workloadmgr.base] Vm details :response: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1', 'x-compute-request-id': 'req-74e5f154-5c6a-401a-8e96-3f60739407fd', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-74e5f154-5c6a-401a-8e96-3f60739407fd'}
Body: {u'server': {u'OS-EXT-STS:task_state': None, u'addresses': {}, u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/v2.1/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1', u'rel': u'self'}, {u'href': u'https://traincentos.triliodata.demo:8774/servers/8276bbc6-621a-44c8-9a44-6a14e726e1d1', u'rel': u'bookmark'}], u'image': {u'id': u'bc2fd9b7-b114-4c75-8350-69aa90223793', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793', u'rel': u'bookmark'}]}, u'OS-EXT-STS:vm_state': u'error', u'OS-EXT-SRV-ATTR:instance_name': u'instance-00000011', u'OS-SRV-USG:launched_at': None, u'flavor': {u'id': u'9265bb6c-4fe7-4dda-a974-1886be221574', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/flavors/9265bb6c-4fe7-4dda-a974-1886be221574', u'rel': u'bookmark'}]}, u'id': u'8276bbc6-621a-44c8-9a44-6a14e726e1d1', u'user_id': u'1aa40866a8ab4c8fba95febee63f4b3f', u'OS-DCF:diskConfig': u'MANUAL', u'accessIPv4': u'', u'accessIPv6': u'', u'OS-EXT-STS:power_state': 0, u'OS-EXT-AZ:availability_zone': u'nova', u'config_drive': u'', u'status': u'ERROR', u'updated': u'2020-05-04T13:39:29Z', u'hostId': u'', u'OS-EXT-SRV-ATTR:host': None, u'OS-SRV-USG:terminated_at': None, u'key_name': u'tempest_test_key_pair', u'OS-EXT-SRV-ATTR:hypervisor_hostname': None, u'name': u'tempest_test_vm_1', u'created': u'2020-05-04T13:39:28Z', u'tenant_id': u'487d69e3dac6411ba4ffca3a5ac61738', u'os-extended-volumes:volumes_attached': [], u'fault': {u'message': u'No valid host was found. ', u'code': 500, u'details': u'Traceback (most recent call last):\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py", line 160, in select_destinations\n    return cctxt.call(ctxt, \'select_destinations\', **msg_args)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py", line 181, in call\n    transport_options=self.transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py", line 129, in _send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 646, in send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py", line 235, in inner\n    return func(*args, **kwargs)\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py", line 199, in select_destinations\n    raise exception.NoValidHost(reason="")\n\nNoValidHost: No valid host was found. \n\n', u'created': u'2020-05-04T13:39:29Z'}, u'metadata': {}}}
2020-05-05 10:36:31,527 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45 0.593s
2020-05-05 10:36:31,528 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45', 'x-compute-request-id': 'req-e9ed59b1-9657-419e-ae24-0ee4a83dcf52', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-e9ed59b1-9657-419e-ae24-0ee4a83dcf52'}
        Body: {"server": {"OS-EXT-STS:task_state": null, "addresses": {}, "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45", "rel": "bookmark"}], "image": {"id": "bc2fd9b7-b114-4c75-8350-69aa90223793", "links": [{"href": "https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793", "rel": "bookmark"}]}, "OS-EXT-STS:vm_state": "error", "OS-EXT-SRV-ATTR:instance_name": "instance-00000010", "OS-SRV-USG:launched_at": null, "flavor": {"id": "294ab694-bd0f-46d9-ab62-1205d4af588a", "links": [{"href": "https://traincentos.triliodata.demo:8774/flavors/294ab694-bd0f-46d9-ab62-1205d4af588a", "rel": "bookmark"}]}, "id": "1bda2d0b-c2e2-4933-8772-5f2b0b68ec45", "user_id": "1aa40866a8ab4c8fba95febee63f4b3f", "OS-DCF:diskConfig": "MANUAL", "accessIPv4": "", "accessIPv6": "", "OS-EXT-STS:power_state": 0, "OS-EXT-AZ:availability_zone": "nova", "config_drive": "", "status": "ERROR", "updated": "2020-05-04T13:39:06Z", "hostId": "", "OS-EXT-SRV-ATTR:host": null, "OS-SRV-USG:terminated_at": null, "key_name": "tempest_test_key_pair", "OS-EXT-SRV-ATTR:hypervisor_hostname": null, "name": "tempest_test_vm_1", "created": "2020-05-04T13:39:05Z", "tenant_id": "487d69e3dac6411ba4ffca3a5ac61738", "os-extended-volumes:volumes_attached": [], "fault": {"message": "No valid host was found. ", "code": 500, "details": "Traceback (most recent call last):\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py\", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py\", line 160, in select_destinations\n    return cctxt.call(ctxt, 'select_destinations', **msg_args)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py\", line 181, in call\n    transport_options=self.transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py\", line 129, in _send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 646, in send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py\", line 235, in inner\n    return func(*args, **kwargs)\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py\", line 199, in select_destinations\n    raise exception.NoValidHost(reason=\"\")\n\nNoValidHost: No valid host was found. \n\n", "created": "2020-05-04T13:39:06Z"}, "metadata": {}}}
2020-05-05 10:36:31,541 22148 DEBUG    [tempest.api.workloadmgr.base] Vm details :response: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45', 'x-compute-request-id': 'req-e9ed59b1-9657-419e-ae24-0ee4a83dcf52', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-e9ed59b1-9657-419e-ae24-0ee4a83dcf52'}
Body: {u'server': {u'OS-EXT-STS:task_state': None, u'addresses': {}, u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/v2.1/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45', u'rel': u'self'}, {u'href': u'https://traincentos.triliodata.demo:8774/servers/1bda2d0b-c2e2-4933-8772-5f2b0b68ec45', u'rel': u'bookmark'}], u'image': {u'id': u'bc2fd9b7-b114-4c75-8350-69aa90223793', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793', u'rel': u'bookmark'}]}, u'OS-EXT-STS:vm_state': u'error', u'OS-EXT-SRV-ATTR:instance_name': u'instance-00000010', u'OS-SRV-USG:launched_at': None, u'flavor': {u'id': u'294ab694-bd0f-46d9-ab62-1205d4af588a', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/flavors/294ab694-bd0f-46d9-ab62-1205d4af588a', u'rel': u'bookmark'}]}, u'id': u'1bda2d0b-c2e2-4933-8772-5f2b0b68ec45', u'user_id': u'1aa40866a8ab4c8fba95febee63f4b3f', u'OS-DCF:diskConfig': u'MANUAL', u'accessIPv4': u'', u'accessIPv6': u'', u'OS-EXT-STS:power_state': 0, u'OS-EXT-AZ:availability_zone': u'nova', u'config_drive': u'', u'status': u'ERROR', u'updated': u'2020-05-04T13:39:06Z', u'hostId': u'', u'OS-EXT-SRV-ATTR:host': None, u'OS-SRV-USG:terminated_at': None, u'key_name': u'tempest_test_key_pair', u'OS-EXT-SRV-ATTR:hypervisor_hostname': None, u'name': u'tempest_test_vm_1', u'created': u'2020-05-04T13:39:05Z', u'tenant_id': u'487d69e3dac6411ba4ffca3a5ac61738', u'os-extended-volumes:volumes_attached': [], u'fault': {u'message': u'No valid host was found. ', u'code': 500, u'details': u'Traceback (most recent call last):\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py", line 160, in select_destinations\n    return cctxt.call(ctxt, \'select_destinations\', **msg_args)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py", line 181, in call\n    transport_options=self.transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py", line 129, in _send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 646, in send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py", line 235, in inner\n    return func(*args, **kwargs)\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py", line 199, in select_destinations\n    raise exception.NoValidHost(reason="")\n\nNoValidHost: No valid host was found. \n\n', u'created': u'2020-05-04T13:39:06Z'}, u'metadata': {}}}
2020-05-05 10:36:31,869 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6 0.326s
2020-05-05 10:36:31,869 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6', 'x-compute-request-id': 'req-949bb172-9d18-4b3e-8786-4f4d6ad49667', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-949bb172-9d18-4b3e-8786-4f4d6ad49667'}
        Body: {"server": {"OS-EXT-STS:task_state": null, "addresses": {}, "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6", "rel": "bookmark"}], "image": {"id": "bc2fd9b7-b114-4c75-8350-69aa90223793", "links": [{"href": "https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793", "rel": "bookmark"}]}, "OS-EXT-STS:vm_state": "error", "OS-EXT-SRV-ATTR:instance_name": "instance-0000000f", "OS-SRV-USG:launched_at": null, "flavor": {"id": "72e9c88d-c19d-4adf-8941-9f75befa48e4", "links": [{"href": "https://traincentos.triliodata.demo:8774/flavors/72e9c88d-c19d-4adf-8941-9f75befa48e4", "rel": "bookmark"}]}, "id": "8ffb8122-3d97-42ff-886f-7b437954d3d6", "user_id": "1aa40866a8ab4c8fba95febee63f4b3f", "OS-DCF:diskConfig": "MANUAL", "accessIPv4": "", "accessIPv6": "", "OS-EXT-STS:power_state": 0, "OS-EXT-AZ:availability_zone": "nova", "config_drive": "", "status": "ERROR", "updated": "2020-05-04T13:38:44Z", "hostId": "", "OS-EXT-SRV-ATTR:host": null, "OS-SRV-USG:terminated_at": null, "key_name": "tempest_test_key_pair", "OS-EXT-SRV-ATTR:hypervisor_hostname": null, "name": "tempest_test_vm_1", "created": "2020-05-04T13:38:44Z", "tenant_id": "487d69e3dac6411ba4ffca3a5ac61738", "os-extended-volumes:volumes_attached": [], "fault": {"message": "No valid host was found. ", "code": 500, "details": "Traceback (most recent call last):\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py\", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py\", line 160, in select_destinations\n    return cctxt.call(ctxt, 'select_destinations', **msg_args)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py\", line 181, in call\n    transport_options=self.transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py\", line 129, in _send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 646, in send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py\", line 235, in inner\n    return func(*args, **kwargs)\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py\", line 199, in select_destinations\n    raise exception.NoValidHost(reason=\"\")\n\nNoValidHost: No valid host was found. \n\n", "created": "2020-05-04T13:38:44Z"}, "metadata": {}}}
2020-05-05 10:36:31,881 22148 DEBUG    [tempest.api.workloadmgr.base] Vm details :response: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6', 'x-compute-request-id': 'req-949bb172-9d18-4b3e-8786-4f4d6ad49667', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-949bb172-9d18-4b3e-8786-4f4d6ad49667'}
Body: {u'server': {u'OS-EXT-STS:task_state': None, u'addresses': {}, u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/v2.1/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6', u'rel': u'self'}, {u'href': u'https://traincentos.triliodata.demo:8774/servers/8ffb8122-3d97-42ff-886f-7b437954d3d6', u'rel': u'bookmark'}], u'image': {u'id': u'bc2fd9b7-b114-4c75-8350-69aa90223793', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793', u'rel': u'bookmark'}]}, u'OS-EXT-STS:vm_state': u'error', u'OS-EXT-SRV-ATTR:instance_name': u'instance-0000000f', u'OS-SRV-USG:launched_at': None, u'flavor': {u'id': u'72e9c88d-c19d-4adf-8941-9f75befa48e4', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/flavors/72e9c88d-c19d-4adf-8941-9f75befa48e4', u'rel': u'bookmark'}]}, u'id': u'8ffb8122-3d97-42ff-886f-7b437954d3d6', u'user_id': u'1aa40866a8ab4c8fba95febee63f4b3f', u'OS-DCF:diskConfig': u'MANUAL', u'accessIPv4': u'', u'accessIPv6': u'', u'OS-EXT-STS:power_state': 0, u'OS-EXT-AZ:availability_zone': u'nova', u'config_drive': u'', u'status': u'ERROR', u'updated': u'2020-05-04T13:38:44Z', u'hostId': u'', u'OS-EXT-SRV-ATTR:host': None, u'OS-SRV-USG:terminated_at': None, u'key_name': u'tempest_test_key_pair', u'OS-EXT-SRV-ATTR:hypervisor_hostname': None, u'name': u'tempest_test_vm_1', u'created': u'2020-05-04T13:38:44Z', u'tenant_id': u'487d69e3dac6411ba4ffca3a5ac61738', u'os-extended-volumes:volumes_attached': [], u'fault': {u'message': u'No valid host was found. ', u'code': 500, u'details': u'Traceback (most recent call last):\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py", line 160, in select_destinations\n    return cctxt.call(ctxt, \'select_destinations\', **msg_args)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py", line 181, in call\n    transport_options=self.transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py", line 129, in _send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 646, in send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py", line 235, in inner\n    return func(*args, **kwargs)\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py", line 199, in select_destinations\n    raise exception.NoValidHost(reason="")\n\nNoValidHost: No valid host was found. \n\n', u'created': u'2020-05-04T13:38:44Z'}, u'metadata': {}}}
2020-05-05 10:36:32,395 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492 0.512s
2020-05-05 10:36:32,396 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492', 'x-compute-request-id': 'req-fd4981b0-70fc-4317-a735-6e2df61580ff', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-fd4981b0-70fc-4317-a735-6e2df61580ff'}
        Body: {"server": {"OS-EXT-STS:task_state": null, "addresses": {}, "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492", "rel": "bookmark"}], "image": {"id": "bc2fd9b7-b114-4c75-8350-69aa90223793", "links": [{"href": "https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793", "rel": "bookmark"}]}, "OS-EXT-STS:vm_state": "error", "OS-EXT-SRV-ATTR:instance_name": "instance-0000000e", "OS-SRV-USG:launched_at": null, "flavor": {"id": "b4e59f1c-a760-458a-ab73-e61fd171645d", "links": [{"href": "https://traincentos.triliodata.demo:8774/flavors/b4e59f1c-a760-458a-ab73-e61fd171645d", "rel": "bookmark"}]}, "id": "7d4e4e4b-4729-4603-88ac-8ccf0794e492", "user_id": "1aa40866a8ab4c8fba95febee63f4b3f", "OS-DCF:diskConfig": "MANUAL", "accessIPv4": "", "accessIPv6": "", "OS-EXT-STS:power_state": 0, "OS-EXT-AZ:availability_zone": "nova", "config_drive": "", "status": "ERROR", "updated": "2020-05-04T13:38:20Z", "hostId": "", "OS-EXT-SRV-ATTR:host": null, "OS-SRV-USG:terminated_at": null, "key_name": "tempest_test_key_pair", "OS-EXT-SRV-ATTR:hypervisor_hostname": null, "name": "tempest_test_vm_1", "created": "2020-05-04T13:38:20Z", "tenant_id": "487d69e3dac6411ba4ffca3a5ac61738", "os-extended-volumes:volumes_attached": [], "fault": {"message": "No valid host was found. ", "code": 500, "details": "Traceback (most recent call last):\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py\", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py\", line 160, in select_destinations\n    return cctxt.call(ctxt, 'select_destinations', **msg_args)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py\", line 181, in call\n    transport_options=self.transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py\", line 129, in _send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 646, in send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py\", line 235, in inner\n    return func(*args, **kwargs)\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py\", line 199, in select_destinations\n    raise exception.NoValidHost(reason=\"\")\n\nNoValidHost: No valid host was found. \n\n", "created": "2020-05-04T13:38:20Z"}, "metadata": {}}}
2020-05-05 10:36:32,413 22148 DEBUG    [tempest.api.workloadmgr.base] Vm details :response: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492', 'x-compute-request-id': 'req-fd4981b0-70fc-4317-a735-6e2df61580ff', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-fd4981b0-70fc-4317-a735-6e2df61580ff'}
Body: {u'server': {u'OS-EXT-STS:task_state': None, u'addresses': {}, u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/v2.1/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492', u'rel': u'self'}, {u'href': u'https://traincentos.triliodata.demo:8774/servers/7d4e4e4b-4729-4603-88ac-8ccf0794e492', u'rel': u'bookmark'}], u'image': {u'id': u'bc2fd9b7-b114-4c75-8350-69aa90223793', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793', u'rel': u'bookmark'}]}, u'OS-EXT-STS:vm_state': u'error', u'OS-EXT-SRV-ATTR:instance_name': u'instance-0000000e', u'OS-SRV-USG:launched_at': None, u'flavor': {u'id': u'b4e59f1c-a760-458a-ab73-e61fd171645d', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/flavors/b4e59f1c-a760-458a-ab73-e61fd171645d', u'rel': u'bookmark'}]}, u'id': u'7d4e4e4b-4729-4603-88ac-8ccf0794e492', u'user_id': u'1aa40866a8ab4c8fba95febee63f4b3f', u'OS-DCF:diskConfig': u'MANUAL', u'accessIPv4': u'', u'accessIPv6': u'', u'OS-EXT-STS:power_state': 0, u'OS-EXT-AZ:availability_zone': u'nova', u'config_drive': u'', u'status': u'ERROR', u'updated': u'2020-05-04T13:38:20Z', u'hostId': u'', u'OS-EXT-SRV-ATTR:host': None, u'OS-SRV-USG:terminated_at': None, u'key_name': u'tempest_test_key_pair', u'OS-EXT-SRV-ATTR:hypervisor_hostname': None, u'name': u'tempest_test_vm_1', u'created': u'2020-05-04T13:38:20Z', u'tenant_id': u'487d69e3dac6411ba4ffca3a5ac61738', u'os-extended-volumes:volumes_attached': [], u'fault': {u'message': u'No valid host was found. ', u'code': 500, u'details': u'Traceback (most recent call last):\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py", line 160, in select_destinations\n    return cctxt.call(ctxt, \'select_destinations\', **msg_args)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py", line 181, in call\n    transport_options=self.transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py", line 129, in _send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 646, in send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py", line 235, in inner\n    return func(*args, **kwargs)\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py", line 199, in select_destinations\n    raise exception.NoValidHost(reason="")\n\nNoValidHost: No valid host was found. \n\n', u'created': u'2020-05-04T13:38:20Z'}, u'metadata': {}}}
2020-05-05 10:36:33,210 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c 0.796s
2020-05-05 10:36:33,210 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c', 'x-compute-request-id': 'req-56ce06ac-9685-49e6-a67b-d50fbccff27c', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-56ce06ac-9685-49e6-a67b-d50fbccff27c'}
        Body: {"server": {"OS-EXT-STS:task_state": null, "addresses": {}, "links": [{"href": "https://traincentos.triliodata.demo:8774/v2.1/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c", "rel": "self"}, {"href": "https://traincentos.triliodata.demo:8774/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c", "rel": "bookmark"}], "image": {"id": "bc2fd9b7-b114-4c75-8350-69aa90223793", "links": [{"href": "https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793", "rel": "bookmark"}]}, "OS-EXT-STS:vm_state": "error", "OS-EXT-SRV-ATTR:instance_name": "instance-0000000d", "OS-SRV-USG:launched_at": null, "flavor": {"id": "929a1216-5ded-452e-a705-8f52e250ab33", "links": [{"href": "https://traincentos.triliodata.demo:8774/flavors/929a1216-5ded-452e-a705-8f52e250ab33", "rel": "bookmark"}]}, "id": "297fffd4-c2a9-4e24-8a93-97efbb3f544c", "user_id": "1aa40866a8ab4c8fba95febee63f4b3f", "OS-DCF:diskConfig": "MANUAL", "accessIPv4": "", "accessIPv6": "", "OS-EXT-STS:power_state": 0, "OS-EXT-AZ:availability_zone": "nova", "config_drive": "", "status": "ERROR", "updated": "2020-05-04T13:37:21Z", "hostId": "", "OS-EXT-SRV-ATTR:host": null, "OS-SRV-USG:terminated_at": null, "key_name": "tempest_test_key_pair", "OS-EXT-SRV-ATTR:hypervisor_hostname": null, "name": "tempest_test_vm_1", "created": "2020-05-04T13:37:21Z", "tenant_id": "487d69e3dac6411ba4ffca3a5ac61738", "os-extended-volumes:volumes_attached": [], "fault": {"message": "No valid host was found. ", "code": 500, "details": "Traceback (most recent call last):\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py\", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py\", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py\", line 160, in select_destinations\n    return cctxt.call(ctxt, 'select_destinations', **msg_args)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py\", line 181, in call\n    transport_options=self.transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py\", line 129, in _send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 646, in send\n    transport_options=transport_options)\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py\", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py\", line 235, in inner\n    return func(*args, **kwargs)\n\n  File \"/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py\", line 199, in select_destinations\n    raise exception.NoValidHost(reason=\"\")\n\nNoValidHost: No valid host was found. \n\n", "created": "2020-05-04T13:37:21Z"}, "metadata": {}}}
2020-05-05 10:36:33,222 22148 DEBUG    [tempest.api.workloadmgr.base] Vm details :response: {'status': '200', 'content-length': '3463', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c', 'x-compute-request-id': 'req-56ce06ac-9685-49e6-a67b-d50fbccff27c', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-56ce06ac-9685-49e6-a67b-d50fbccff27c'}
Body: {u'server': {u'OS-EXT-STS:task_state': None, u'addresses': {}, u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/v2.1/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c', u'rel': u'self'}, {u'href': u'https://traincentos.triliodata.demo:8774/servers/297fffd4-c2a9-4e24-8a93-97efbb3f544c', u'rel': u'bookmark'}], u'image': {u'id': u'bc2fd9b7-b114-4c75-8350-69aa90223793', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/images/bc2fd9b7-b114-4c75-8350-69aa90223793', u'rel': u'bookmark'}]}, u'OS-EXT-STS:vm_state': u'error', u'OS-EXT-SRV-ATTR:instance_name': u'instance-0000000d', u'OS-SRV-USG:launched_at': None, u'flavor': {u'id': u'929a1216-5ded-452e-a705-8f52e250ab33', u'links': [{u'href': u'https://traincentos.triliodata.demo:8774/flavors/929a1216-5ded-452e-a705-8f52e250ab33', u'rel': u'bookmark'}]}, u'id': u'297fffd4-c2a9-4e24-8a93-97efbb3f544c', u'user_id': u'1aa40866a8ab4c8fba95febee63f4b3f', u'OS-DCF:diskConfig': u'MANUAL', u'accessIPv4': u'', u'accessIPv6': u'', u'OS-EXT-STS:power_state': 0, u'OS-EXT-AZ:availability_zone': u'nova', u'config_drive': u'', u'status': u'ERROR', u'updated': u'2020-05-04T13:37:21Z', u'hostId': u'', u'OS-EXT-SRV-ATTR:host': None, u'OS-SRV-USG:terminated_at': None, u'key_name': u'tempest_test_key_pair', u'OS-EXT-SRV-ATTR:hypervisor_hostname': None, u'name': u'tempest_test_vm_1', u'created': u'2020-05-04T13:37:21Z', u'tenant_id': u'487d69e3dac6411ba4ffca3a5ac61738', u'os-extended-volumes:volumes_attached': [], u'fault': {u'message': u'No valid host was found. ', u'code': 500, u'details': u'Traceback (most recent call last):\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 1333, in schedule_and_build_instances\n    instance_uuids, return_alternates=True)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/conductor/manager.py", line 839, in _schedule_instances\n    return_alternates=return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/client/query.py", line 42, in select_destinations\n    instance_uuids, return_objects, return_alternates)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/rpcapi.py", line 160, in select_destinations\n    return cctxt.call(ctxt, \'select_destinations\', **msg_args)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/client.py", line 181, in call\n    transport_options=self.transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/transport.py", line 129, in _send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 646, in send\n    transport_options=transport_options)\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py", line 636, in _send\n    raise result\nNoValidHost_Remote: No valid host was found. \nTraceback (most recent call last):\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/oslo_messaging/rpc/server.py", line 235, in inner\n    return func(*args, **kwargs)\n\n  File "/openstack/venvs/nova-20.0.2.dev6/lib/python2.7/site-packages/nova/scheduler/manager.py", line 199, in select_destinations\n    raise exception.NoValidHost(reason="")\n\nNoValidHost: No valid host was found. \n\n', u'created': u'2020-05-04T13:37:21Z'}, u'metadata': {}}}
2020-05-05 10:36:33,223 22148 DEBUG    [tempest.api.workloadmgr.test_cleaner] 
vms deleted

2020-05-05 10:36:33,434 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8776/v2/487d69e3dac6411ba4ffca3a5ac61738/volumes 0.210s
2020-05-05 10:36:33,434 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '15', 'content-location': 'https://traincentos.triliodata.demo:8776/v2/487d69e3dac6411ba4ffca3a5ac61738/volumes', 'x-compute-request-id': 'req-a4de9370-32ab-4142-8cd7-42afbd116cbb', 'connection': 'close', 'content-type': 'application/json', 'x-openstack-request-id': 'req-a4de9370-32ab-4142-8cd7-42afbd116cbb'}
        Body: {"volumes": []}
2020-05-05 10:36:33,435 22148 DEBUG    [tempest.api.workloadmgr.test_cleaner] 
volumes deleted

2020-05-05 10:36:33,553 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 200 GET https://traincentos.triliodata.demo:8774/v2.1/os-keypairs 0.117s
2020-05-05 10:36:33,553 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '200', 'content-length': '16', 'content-location': 'https://traincentos.triliodata.demo:8774/v2.1/os-keypairs', 'x-compute-request-id': 'req-8ec77ef5-5b2a-4e86-9ae7-1416fd9a8f5c', 'vary': 'OpenStack-API-Version, X-OpenStack-Nova-API-Version', 'openstack-api-version': 'compute 2.1', 'connection': 'close', 'x-openstack-nova-api-version': '2.1', 'content-type': 'application/json', 'x-openstack-request-id': 'req-8ec77ef5-5b2a-4e86-9ae7-1416fd9a8f5c'}
        Body: {"keypairs": []}
2020-05-05 10:36:33,557 22148 DEBUG    [tempest.api.workloadmgr.test_cleaner] 
keypairs deleted

2020-05-05 10:36:33,736 22148 DEBUG    [tempest.api.workloadmgr.base] Response:{"policy_list": []}
2020-05-05 10:36:33,736 22148 DEBUG    [tempest.api.workloadmgr.test_cleaner] 
policies deleted

2020-05-05 10:36:33,848 22148 INFO     [tempest_lib.common.rest_client] Request (WorkloadTest:test_cleaner): 404 GET https://traincentos.triliodata.demo:9696/v2.0/networks/180984ff-2dcd-47ab-882e-2ae6dbca4d36 0.111s
2020-05-05 10:36:33,848 22148 DEBUG    [tempest_lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'status': '404', 'content-length': '138', 'connection': 'close', 'date': 'Tue, 05 May 2020 05:06:33 GMT', 'content-type': 'application/json', 'x-openstack-request-id': 'req-045d171e-3669-4df4-be3f-7e0771998edb'}
        Body: {"NeutronError": {"message": "Network 180984ff-2dcd-47ab-882e-2ae6dbca4d36 could not be found.", "type": "NetworkNotFound", "detail": ""}}
2020-05-05 10:36:33,849 22148 ERROR    [tempest.api.workloadmgr.test_cleaner] Exception: Object not found
Details: {u'message': u'Network 180984ff-2dcd-47ab-882e-2ae6dbca4d36 could not be found.', u'type': u'NetworkNotFound', u'detail': u''}
0
Content-Type: text/plain;charset=utf8
stderr
C45
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
/home/nchavan/tempest/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:986: InsecureRequestWarning: Unverified HTTPS request is being made to host 'osavip-nc-168.triliodata.demo'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
0
]
tags: -worker-0
